webapp/src/shared/components/Avatar/EditableAvatar.css

```diff
@@ -20,11 +25,10 @@
 }
 
 .editable-avatar__image {
-  user-drag: none;
   -webkit-user-drag: none;
   margin: 0;
-  width: 16rem;
-  height: 16rem;
+  width: 15rem;
```

Comment:
This is veeeeery tricky to change. But I understand where is this coming from. I'm not sure what's wrong with my parameters. I've fixed them twice, and I think there is, again, some problem with the scale of the image and the offset of the picturePosition, for each of the different avatar sizes. Maybe someone has some idea.

Reply:
Yes, this was me playing around with sizes, I have restored the values. Thanks for pointing out